### DISCUSSION TITLE
creating a gui after getting input from the first gui

### DISCUSSION THREAD

**Discussion by Ms-Ajith** [chriskiehl/Gooey#844] [General]
Hello,
I have created a working gui and would like to create a second one based on some inputs from the first.

For example. If user choose yes in the first gui interface, a new gui should pop up which asks for some other input. How can I achieve this? Are you having some examples for this?

Also Is it possible to change the Start and Cancel default button names to some other names?

**Comment by chriskiehl** [comment]
Not by default. Gooey isn't really aiming at being a general terminal replacement. However, you can still fake it by using some of the recommendations from @eladeyal-intel's in this issue: https://github.com/chriskiehl/Gooey/issues/222

**Reply by Ms-Ajith** [reply]
Thanks alot for the help :) 👍
